scripts/partner_batch_imports.py

```diff
@@ -82,6 +83,8 @@
     "/openlibrary-client/master/olclient/schemata/import.schema.json"
 )
 
+required_fields = []
```

Comment:
```suggestion
required_fields: list[str] = []
```
Let's try this again....